### PR TITLE
allow ns-process-data to use CR2 raw images

### PR DESCRIPTION
**Pull request by machenmusik**
(Note, this may allow other raw image formats to work as well, but I only have some CR2 that I can't share at present.)

### CODE REVIEW

File: nerfstudio/process_data/process_data_utils.py [hunk 1]
```diff
@@ -207,7 +210,15 @@ def copy_images_list(
             CONSOLE.log(f"Copying image {idx + 1} of {len(image_paths)}...")
         copied_image_path = image_dir / f"frame_{idx + 1:05d}{image_path.suffix}"
         try:
-            shutil.copy(image_path, copied_image_path)
+            # if CR2 raw, we want to read raw and write TIFF, and change the file suffix for downstream processing
+            if str(image_path).lower().endswith(".cr2"):
```
Comment (tancik) [inline nerfstudio/process_data/process_data_utils.py:214]: `image_path.suffix().lower() == ".cr2"`

**Review by tancik — COMMENTED** [review]
Seems worthwhile, thanks for adding. Added a minor comment.

**Comment by machenmusik** [comment]
Do others have non-Canon raw image samples that should also be supported?  CR2 is one of the Canon suffixes, but others such as Nikon and Sony should work as well (in theory).

**Comment by machenmusik** [comment]
Made it easy to add other suffixes to treat as raw images.

**Review by tancik — APPROVED** [review]
LGTM

**Pull request merged**